platform/linux-generic/include/odp_config_internal.h

```diff
@@ -22,7 +22,7 @@ extern "C" {
 /*
  * Maximum number of pools.
  */
-#define ODP_CONFIG_POOLS 32
+#define CONFIG_POOLS 32
```

Comment:
Internally "public" so use the `_ODP` prefix in these?

Reply:
I'd prefer the maintain the current naming scheme for now as this causes less code changes (3 vs. 24 renames).